nix/purescript.nix

```diff
@@ -36,9 +36,17 @@ let
   packagesJson = "${src}/packages.json";
 
   # remove any files that have appeared in local builds
-  cleanSrcs = builtins.filterSource (path: type: !(pkgs.lib.elem (baseNameOf path)
-                                                  [".spago" ".spago2nix" "generated" "generated-docs" "output" "dist"]))
-                                                  src;
+  # cleanSrcs = builtins.filterSource (path: type: !(pkgs.lib.elem (baseNameOf path)
```

Comment:
In or out?

Reply:
I kind of wanted to leave it in for when we look at this in another PR...